templates/solutions/ubuntu-os/index.html

```diff
@@ -248,8 +248,11 @@ <h2>Everywhere and for everyone</h2>
     </div>
     <div class="row p-section--shallow">
       <div class="col-9 col-start-large-4">
-        <div class="p-equal-height-row has-divider-1 has-divider-3">
+        <div class="p-equal-height-row has-divider-3">
```

Comment:
```suggestion
        <div class="p-equal-height-row--wrap has-divider-3">
```
based on what mattea says, could make this a `--wrap` element